src/main/webapp/jsp/fp/RecurringDisbursementVoucher.jsp

```diff
@@ -0,0 +1,105 @@
+<%--
```

Comment:
Since there is so much overlap between this file and DisbursementVoucher.jsp, can we come up with a way to make some of this more generic, promote reuse and avoid the duplication?